bucket/jcpicker.json

```diff
@@ -0,0 +1,36 @@
+{
+    "homepage": "https://annystudio.com/software/colorpicker/",
+    "description": "Free portable offline colour picker and colour editor for web designers and digital artists.",
+    "version": "5.5",
+    "license": "Freeware",
```

Comment:
URL?

Reply:
The author does not provide a URL for their license. They just mention "Freeware".